lib/engines/object_store/config/routes.rb

```diff
@@ -0,0 +1,10 @@
+ObjectStore::Engine.routes.draw do
+  namespace :object_store, path: "/" do
+    root to: "content_blocks#index", via: :get
+
+    resources :content_blocks do
+      get "new/:block_type", to: "content_blocks#new", on: :collection, as: :new
+      get ":block_type/schema", to: "content_blocks#info", on: :collection, as: :info
+    end
```

Comment:
I think I'd prefer the URLs to look like this:

```
object_store/content_blocks # Show all content blocks
object_store/content_blocks/new # Create a new content block (ask what block type is required)
object_store/content_blocks/:object_type/new # Render the form for a new :object_type
object_store/content_blocks/:object_type # Create an :object_type
etc etc
```

I think we can do this like so:

```
    resources :content_blocks do
      scope ":block_type" do
        resources :content_blocks, path: "/"
      end
    end
```